workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/Config/LocatorConfigTests.cs

```diff
@@ -110,21 +144,21 @@ public void CreateConnectionConfigFromCommandLine_should_parse_correctly()
         }
 
         [Test]
-        public void CreateConnectionConfigFromCommandLine_should_provide_defaults_for_missing_values()
+        public void CreateConnectionConfigFromCommandLine_should_provide_default_values()
         {
             var parsedArgs = new Dictionary<string, string>();
             var config = LocatorConfig.CreateConnectionConfigFromCommandLine(parsedArgs);
 
-            Assert.AreEqual(LocatorCredentialsType.LoginToken, config.LocatorParameters.CredentialsType);
-            Assert.AreEqual(string.Empty, config.LocatorParameters.LoginToken.Token);
             Assert.AreEqual(RuntimeConfigDefaults.LocatorHost, config.LocatorHost);
-            Assert.AreEqual(string.Empty, config.LocatorParameters.ProjectName);
             Assert.AreEqual(RuntimeConfigDefaults.LinkProtocol, config.LinkProtocol);
         }
 
         private static LocatorConfig GetDefaultWorkingConfig()
         {
-            return new LocatorConfig();
+            var locatorConfig = new LocatorConfig();
```

Comment:
that method name is now misleading. would it be better to say `GetDefaultConfigWithLoginToken`?

Reply:
Renamed!